workflows/postprocessing.nf

```diff
@@ -137,16 +137,28 @@ process writemeta{
     """
 }
 
-// We assume that the input gvcf files are indexed
-def replicate_excludemnps_output_format(input_channel) {
-    def with_tbi = input_channel.filter{meta, vcf -> file(vcf + ".tbi").exists()}
-        .map{meta, vcf -> [meta, [file(vcf), file(vcf + ".tbi")]]}
 
-    def tbi_input = input_channel.filter{meta, vcf -> !file(vcf + ".tbi").exists()}
-    def tbi_output = initial_tabix(tbi_input)
-    def with_generated_tbi = tbi_input.join(tbi_output).map{meta, vcf, tbi -> [meta, [vcf, tbi]]}
+def exclude_mnps(input_channel, do_exclude_mnps) {
```

Comment:
I think we now have 3 entities named exlude_mnps: a parameter, a function and a process. Maybe this could be renamed?

Reply:
Good point.  I will rename to handle_mnps.

Reply:
Done